Better launcher animation

bomberfish opened the pull request:
Makes launcher opening animation smoother and slightly longer

TODO: Animate icons like CrOS?

Old:

[2024-03-25_10-03-1711375924.webm](https://github.com/MercuryWorkshop/anuraOS/assets/87151697/aee99e0e-9f64-40d1-99ef-8eb88cebf7c8)

New:

[2024-03-25_10-03-1711375908.webm](https://github.com/MercuryWorkshop/anuraOS/assets/87151697/e7c55695-d443-4ac2-b99b-09aba567d631)

bomberfish committed:
better animation

MadjikDotPng commented:
I remember changing this and rafflesia getting mad about it being too slow
This looks good though.

ProgrammerIn-wonderland commented:
These videos you're uploading are not playing for me

ProgrammerIn-wonderland commented:
VLC, discord, chromium, nothing can play these

ProgrammerIn-wonderland commented:
Re-encoded the video through ffmpeg, 

It looks good but it def feels slower

bomberfish committed:
Update Launcher.tsx

bomberfish commented:
Try now, should be the same duration as original but with a smoother timing curve

bomberfish committed:
nicer animation with app grid animation

bomberfish commented:
Added the aformentioned icon animation (and changed the duration again)

[2024-03-26_08-03-1711457281.webm](https://github.com/MercuryWorkshop/anuraOS/assets/87151697/fede400d-d6b8-4057-9bd4-8a3694c9b9ca)

Percslol commented:
Looks good, waiting for @ProgrammerIn-wonderland to review.

ProgrammerIn-wonderland commented:
Yeah

ProgrammerIn-wonderland commented:
It's good

MadjikDotPng commented:
Please merge this.

MadjikDotPng commented:
Oh wait it was already merged

ProgrammerIn-wonderland commented:
Unmerging because koaku commented

5VER5GEUN1X commented:
certified koaku commented unmerge
